pcl_ros/CMakeLists.txt

```diff
@@ -214,6 +215,21 @@ ament_target_dependencies(combined_pointcloud_to_pcd_lib
   tf2_eigen
   tf2_ros)
 
+add_library(bag_to_pcd_lib SHARED tools/bag_to_pcd.cpp)
+target_link_libraries(bag_to_pcd_lib
+  ${PCL_LIBRARIES})
+target_include_directories(bag_to_pcd_lib PUBLIC
+  ${PCL_INCLUDE_DIRS})
+ament_target_dependencies(bag_to_pcd_lib
+  rclcpp
+  rclcpp_components
+  sensor_msgs
+  pcl_conversions
+  rosbag2_transport)
+rclcpp_components_register_node(bag_to_pcd_lib
+  PLUGIN "pcl_ros::BagToPCD"
+  EXECUTABLE bag_to_pcd)
+
```

Comment:
Can you make sure to also install `bag_to_pcd_lib`? I'm getting the following error:
```
ros2 run pcl_ros bag_to_pcd 
terminate called after throwing an instance of 'class_loader::LibraryLoadException'
  what():  Could not load library dlopen error: libbag_to_pcd_lib.so: cannot open shared object file: No such file or directory, at ./src/shared_library.c:99
[ros2run]: Aborte
```